aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -265,6 +269,10 @@ module AptosFramework::Coin {
         amount: u64,
         _cap: &MintCapability<CoinType>,
     ): Coin<CoinType> acquires CoinInfo {
+        if (amount == 0) {
```

Comment:
this too

Reply:
Aborting here also makes sense.

Reply:
Same as previous comment. Abort actually doesn't make sense here as mint is called as part of staking reward distribution. We should avoid aborting if possible but we also don't want to emit an event unnecessarily here.